database/migrations/2022_02_22_181909_create_meetups_table.php

```diff
@@ -12,6 +12,7 @@ public function up(): void
         Schema::create("meetups", function (Blueprint $table): void {
             $table->id();
             $table->foreignId("user_id")->constrained()->onDelete("cascade");
+            $table->foreignId("organization_id")->nullable()->constrained()->onDelete("set null");
```

Comment:
Okay, for now it's okay, but after first deploy the common sense rule is not to edit migration files.